pkgs/tools/networking/xray/default.nix

```diff
@@ -1,26 +1,26 @@
 { lib
 , fetchFromGitHub
 , symlinkJoin
-, buildGoModule
+, buildGo122Module
```

Comment:
this should be done in all-packages as per https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults

Reply:
Tested in master:

```
nix-repl> pkgs = legacyPackages.x86_64-linux

nix-repl> pkgs.buildGoModule == pkgs.buildGo122Module 
true

nix-repl> pkgs.buildGoModule == pkgs.buildGo121Module 
false
```

Maybe the author is on an older fork.